BRLS/src/library/returnBook/ReturnItemUI.java

```diff
@@ -19,43 +19,42 @@ public void run() {
         while (true) {
             
             switch (uiState) {
-            
-            case INITIALISED:
+                case INITIALISED:
                 break;
```

Comment:
PR3 D11
Minor complaint, but the `break;` is nested within the `case INITIALISED`, and so must be indented by 4 spaced, according to Rule 9 of the ITC205 Code Style Guidelines

Reply:
Done